cypress/e2e/tests/home.cy.ts

```diff
@@ -0,0 +1,30 @@
+/********************************************************************************
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation
```

Comment:
Year of file creation needs to be in the header
```suggestion
 * Copyright (c) 2025 Contributors to the Eclipse Foundation
```